cfgov/ask_cfpb/models/django.py

```diff
@@ -148,10 +156,18 @@ def __str__(self):
         return "{} {}".format(self.id, self.slug)
 
     @property
+    def english_page(self):
+        return self.answer_pages.filter(language='en').first()
+
+    @property
+    def spanish_page(self):
+        return self.answer_pages.filter(language='es').first()
+
+    @property
     def available_subcategories(self):
         subcats = []
         for parent in self.category.all():
-            subcats += list(parent.subcategory_set.all())
+            subcats += list(parent.subcategories.all())
```

Comment:
I think you could simplify this with something like:

```
subcats = SubCategory.objects.filter(parent__in=self.category.all())
```

Edit: Actually, just re-use `available_subcategory_qs` below.